.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,16 @@
--   id: prettier
-    name: prettier
-    entry: prettier --write
-    language: node
-    # From https://github.com/prettier/prettier/blob/7a7eb170/docs/index.md
-    files: \.(css|less|scss|ts|tsx|graphql|gql|json|js|jsx|md)$
+- id: prettier
+  name: prettier
+  entry: prettier --write
+  language: node
+  files: "\\.(\
+    css|less|scss\
+    |graphql|gql\
+    |js\
+    |json\
+    |jsx\
```

Comment:
one last thing.. js and jsx could be grouped as well :) no need to merge in one commit.. we are squashing anyway

Reply:
Done :)